### PR TITLE
[Wallet] Various onboarding fixes

### LINKED ISSUE #4164: Nothing happen when user taps on "<" from Select language screen after launching the fresh installed app
**Frequency:** 100%
** Repro App version:** Celo Integration Wallet build v1.8.2 (39)
**Does not repro ON:** App store build v1.8.2 
**Repro on device:**iPhone XS max (13.1), iphone 6+ (12.4.5),iphone 11(13.3)

**Repro Steps:** 
1) Launch the fresh installed app 
2) Make sure Select language page is open
3) Click on < arrow top left corner on Select Language screen 
4) Observe nothing happen when user taps on < arrow. 

**Current Behavior:* Nothing happen when user taps  on < arrow
**Expected Behavior:** As it is first screen of application < arrow should not display. 
**Investigation:** back arrow is work on Select language screen when user open it from setting 
**Impact** Bad user experience
**Attachment:** BackArrowissue.png             
BackArrowIssue

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

- Fix insufficient fund error briefly shown while fetching the balance after enter the invide code
- Hide back button when no back action is possible
- Tweak VerificationLoadingScreen and Interstitial with new colors and fonts
- Update onboarding colors from latest design changes
- Fix nav bar visible in the onboarding success screen

### Tested

Yes ;-)

### Related issues

- #3645
- Fixes #4164

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx [hunk 1]
```diff
@@ -43,7 +44,7 @@ class VerificationInterstitialScreen extends React.Component<WithTranslation> {
             <AnimatedCircle />
             <AnimatedCircle />
           </View>
-          <Text style={fontStyles.h1}>{t('interstitial.header')}</Text>
+          <Text style={fontStyles.h2}>{t('interstitial.header')}</Text>
```
Comment (tarikbellamine) [inline packages/mobile/src/verify/VerificationInterstitialScreen.tsx:47]: nit: Define in stylesheet?

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx [hunk 2]
```diff
@@ -19,7 +20,7 @@ const AnimatedCircle = () => (
 )
 
 class VerificationInterstitialScreen extends React.Component<WithTranslation> {
-  static navigationOptions = { gestureEnabled: false, header: null }
+  static navigationOptions = noHeaderGestureDisabled
```
Comment (tarikbellamine) [inline packages/mobile/src/verify/VerificationInterstitialScreen.tsx:23]: Does defining navigation options like this still work as expected?

**Review by tarikbellamine — APPROVED** [review]
Good stuff! Left small questions but think this is pretty much good to go once the tests are passing

File: packages/mobile/src/verify/VerificationEducationScreen.tsx [hunk 3]
```diff
@@ -33,14 +33,22 @@ type Props = ScreenProps
...
   const userBalance = useSelector((state) => state.stableToken.balance)
   const balanceIsSufficient = isUserBalanceSufficient(userBalance, VERIFICATION_FEE_ESTIMATE)
+  // For now only show error if user has pressed start
+  // with the idea being that by the time the user is done reading the screen the balance will already be known
+  const showError = hasPressedStart && !balanceIsSufficient
   const dispatch = useDispatch()
   const headerHeight = useHeaderHeight()
   const insets = useSafeArea()
 
   function onPressStart() {
```
Comment (jmrossy) [inline packages/mobile/src/verify/VerificationEducationScreen.tsx]: Nit: These should be arrow functions

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationInterstitialScreen.tsx:47]: @tarikbellamine There's not reason to redefine this in the stylesheet if we don't deviate from the predefined value in fontStyles.

File: packages/mobile/src/verify/VerificationLoadingScreen.tsx [hunk 4]
```diff
@@ -123,8 +126,8 @@ class VerificationLoadingScreen extends React.Component<Props> {
           <View style={styles.buttonCancelContainer}>
             <CancelButton onCancel={this.onCancel} />
           </View>
+          <DevSkipButton nextScreen={Screens.VerificationInterstitialScreen} />
```
Comment (jmrossy) [inline packages/mobile/src/verify/VerificationLoadingScreen.tsx]: I can't remember if this is my bug or someone else's but this screen shouldn't have a DevSkipButton. It won't work as expected with verification ongoing in the background. If devs want to skip they should just choose skip on the education screen

**Review by jmrossy — APPROVED** [review]
LGTM! A few small comments below :)

File: packages/mobile/src/verify/VerificationInterstitialScreen.tsx (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/src/verify/VerificationInterstitialScreen.tsx:23]: Yes, tested successfully.

File: packages/mobile/src/verify/VerificationLoadingScreen.tsx (hunk 4 above)
Comment (jeanregisser) [reply packages/mobile/src/verify/VerificationLoadingScreen.tsx]: Makes sense yes, removing then 👍

File: packages/mobile/src/verify/VerificationEducationScreen.tsx (hunk 3 above)
Comment (jeanregisser) [reply packages/mobile/src/verify/VerificationEducationScreen.tsx]: Yes 👍

**Pull request merged**